src/main/java/team/themoment/hellogsmv3/domain/application/controller/ApplicationController.java

```diff
@@ -92,4 +95,11 @@ public CommonApiMessageResponse updateStatus(
         updateApplicationStatusService.execute(applicantId, applicationStatusReqDto);
         return CommonApiMessageResponse.success("수정되었습니다.");
     }
+
+    @GetMapping("/admission-tickets")
+    public List<AdmissionTicketsResDto> findAdmissionTickets(
+    ) {
+        List<AdmissionTicketsResDto> admissionTicketsResDto = queryAdmissionTicketsService.execute();
+        return admissionTicketsResDto;
```

Comment:
이 부분을 인라인화 시켜서 더 깔끔하게 표현할 수 있을것 같아요
```suggestion
        return queryAdmissionTicketsService.execute();
```

Reply:
반영하였습니다.

[73c1e9d](https://github.com/themoment-team/hellogsm-server-v3/pull/33/commits/73c1e9d17fe9e425a5075b09a56d2f5dfc40c2bb)